efel/cppcore/LibV1.cpp

```diff
@@ -181,7 +181,18 @@ int LibV1::ISI_values(mapStr2intVec& IntFeatureData,
     GErrorStr += "\n Three spikes required for calculation of ISI_values.\n";
     return -1;
   }
-  for (size_t i = 2; i < pvTime.size(); i++)
+
+  int IgnoreFirstSpike;
+  vector<int> retIgnore;
+  retVal = getIntParam(IntFeatureData, "ignore_first_spike", retIgnore);
+  if (retVal && (retIgnore.size() > 0) && (retIgnore[0] == 0)) {
```

Comment:
if something goes wrong with the ignore_first_spike feature, retVal will be -1 I think? I assume this will evaluate to True in C++. Is that what we really want? Shouldn't it fall back to the old behavior in that case, i.e. IgnoreFirstSpike = 1 ?

Reply:
Ah, but the size might be 0 in that case. It's still a bit implicit though. Maybe better retVal == 1?

Reply:
Indeed